spec/system/distribution_system_spec.rb

```diff
@@ -395,6 +395,18 @@
       expect(@request.reload).to be_status_fulfilled
     end
 
+    it "copy over relevent request information into the distrubution" do
```

Comment:
@grantmca I think we can remove this test now, correct?

Reply:
Yes! Sorry, must of missed adding the file, Thank you for catching this